pkg/http/handler/timeout_test.go

```diff
@@ -256,16 +256,15 @@ func TestIdleTimeoutHandler(t *testing.T) {
 		firstByteTimeout: longFirstByteTimeout,
 		handler: func(*sync.Mutex, chan error) http.Handler {
 			return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-				w.Write([]byte(""))
+				w.Write([]byte("foo"))
 				time.Sleep(shortIdleTimeout * 2 / 3)
-				w.Write([]byte(""))
+				w.Write([]byte("bar"))
```

Comment:
do we need similar changes in "can still timeout after a few successful writes" or is that one ok?

Reply:
kinda, yes. Updated it to showcase the behavior.